cmd/buildah/bud.go

```diff
@@ -233,5 +233,6 @@ func budCmd(c *cli.Context) error {
 		options.ReportWriter = ioutil.Discard
 	}
 
-	return imagebuildah.BuildDockerfiles(getContext(), store, options, dockerfiles...)
+	_, _, err = imagebuildah.BuildDockerfiles(getContext(), store, options, dockerfiles...)
```

Comment:
Podman calls this, when we merge this there in we'll need to adjust calls there too @mheon